luin/ioredis
# fix: update to latest profile for Redis Cloud

Closes #1634.

This ticket updates the built-in profile the reflect the changes on Redis Cloud side. The CA should work for both Fixed plan and Flexible plan.

To test:

Make sure the following code is able to connect to a Redis Cloud server:

```js
const Redis = require("./built");

const redis = new Redis({
  host: host,
  port: port,
  username: username,
  password: password,
  tls: "RedisCloudFlexible",
});

redis.get("foo", console.log);
```
